src/layman/map/rest_workspace_test.py

```diff
@@ -38,7 +38,7 @@
 
 METADATA_PROPERTIES_EQUAL = METADATA_PROPERTIES
 
-num_maps_before_test = 0
+NUM_MAPS_BEFORE_TEST = 0
```

Comment:
As it is variable and not constant, I prefer lower case and `# pylint: disable=invalid-name`